### DISCUSSION TITLE
FYI: Tapo App for Android Version 3.8.103 not available yet.

### DISCUSSION THREAD

**Discussion by NikosSprocket** [JurajNyiri/HomeAssistant-Tapo-Control#752] [Discuss]
I just checked my Android phone and the latest Tapo version available is 3.7.811. I re-verified on Google Play and that's the only version available at this moment.

**Comment by JurajNyiri** [comment]
Some users were able to get the new version via beta, some others still need to wait. They are probably releasing the new version in staged rollout so you might need to wait a few days, hopefully not more than a week or 2.

Alternative is using iOS device (temporarily) to switch this if you cannot wait.

**Reply by easaw** [reply]
I tried via beta, no luck. Might have to try iOS until then, but it's great news because I have had so many issues! (as I'm sure everyone did).

Thanks @JurajNyiri looking forward to this for Android so we can use your v6 update! :D

**Reply by RoSulek1** [reply]
maybe someone who has access to the latest version could share the APK file (via APKCombo)

**Comment by easaw** [comment]
Ha! I just posted about this 5mins ago, oops!

3.8.103 is iOS - https://apps.apple.com/us/app/tp-link-tapo/id1472718009

Android is not updated yet.

Maybe Juraj can clarify that in the HACS update message ;)

**Reply by JurajNyiri** [reply]
<img width="1197" alt="Screenshot 2024-12-14 at 02 59 25" src="https://github.com/user-attachments/assets/658b498c-d7f5-4d85-95fb-cc754f993321" />
Added!

**Comment by NikosSprocket** [comment]
Thank you all for the info. I will wait for the Android version released via Google.
